src/pages/options/options.js

```diff
@@ -185,8 +185,8 @@ const OptionsFooter = styled.footer`
 const OptionsApp = () => {
   const [sitesTwitter, setSitesTwitter] = useState(getBool(getSetting('sites_twitter')))
   const [darkMode] = useState(getBool(getSetting('darkMode')))
-  const [accessToken] = useState(getSetting('access_token'))
-  const [userName] = useState(getSetting('username'))
+  // const [accessToken] = useState(getSetting('access_token'))
+  // const [userName] = useState(getSetting('username'))
```

Comment:
These are causing issues and make it so the page wouldn't render. I need to figure out what is up with them to make the login/logout links below work, but for the sake of time and not blocking others I just commented them out for now to get the page rendering.

Reply:
Also @anthony-liddle if you have any ideas why this might be causing issues, I'm all 👂👂❗

Reply:
I'm not sure either, but I'll happily take a look!

Reply:
These are async functions ... you need to await their returns.

Reply:
Well there you have it!